source/localizable/models/finding-records.md

```diff
@@ -69,22 +69,42 @@ this.get('store').query('person', {
 
 ### Querying for A Single Record
 
-If you know your query will return only one result Ember Data provides
-a convenience method that will return a promise that resolves with a
-single record. Calling
-[`store.queryRecord()`](http://emberjs.com/api/data/classes/DS.Store.html#method_queryRecord)
-will make a `GET` request with the passed object serialized as query params.
+If you are using an adapter that supports server requests capable of returning a single model object, Ember Data provides
+a convenience method [`store.queryRecord()`](http://emberjs.com/api/data/classes/DS.Store.html#method_queryRecord) that will return a promise that resolves with that single record. The request is made via a method of the same name defined on the adapter.
```

Comment:
I'm ok with that, though how about "The request is made via a method `queryRecord()` defined by the adapter."? 

What I'm trying to give a little emphasis to is just that a custom method does need to be defined; the example in the [API docs](http://emberjs.com/api/data/classes/DS.Adapter.html#method_queryRecord) and indeed the default implementation of `queryRecord()` in the adapter base class are actually flawed since they use pluralized URLs created by `buildURL()`. However I'm happy to accept your judgment on this.